botocore/data/route53/2013-04-01/paginators-1.json

```diff
@@ -14,6 +14,19 @@
       "limit_key": "MaxItems",
       "result_key": "HostedZones"
     },
+    "ListHostedZonesByName": {
+      "more_results": "IsTruncated",
+      "limit_key": "MaxItems",
+      "result_key": "HostedZones",
+      "input_token": [
+        "DNSName",
+        "HostedZoneId"
+      ],
+      "output_token": [
+        "NextDNSName",
```

Comment:
Is this a common standard name for tokens? - its a bit of a bug bare of mine that AWS already has NextToken nextToken and NextMarker for some other API calls - would be good if this were consistent.